staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -107,6 +107,17 @@ func ValidateLabelName(labelName string, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+// ValidateLabelValues validates that the label value is correctly defined.
+func ValidateLabelValues(labelValues []string, fldPath *field.Path) field.ErrorList {
```

Comment:
This is not generic enough to live here, IMO.  It should be defined near where it is used.